src/cnxml/xml.js

```diff
@@ -71,6 +75,9 @@ class Renderer {
 
     renderElement(element) {
         const ns = element.props.xmlns || this.ns
+        if (element.type.toString() == 'Symbol(react.fragment)') {
+            return Array.from(element.props.children).map(c => this.renderComponent(c))
+        }
```

Comment:
`renderElement` accepts only React elements. If you want to add support for fragments then `renderComponent` is the place to do it.

Never case on a debug representation. While in this case it is stable (mandated by the ECMAScript standard), this check is nonetheless incorrect: while the correct symbol for fragments is (probably) `Symbol.for('react.fragment')` this check will also accept every `Symbol('react.fragment')`, which are different symbols. Note ‘are’: Symbol's constructor returns a new symbol every time it's called, regardless of arguments provided. To put it in code:  is false.